packages/web-components/src/color/recipes/neutral-fill-layer.spec.ts

```diff
@@ -11,7 +11,8 @@ describe('neutralFillLayer', (): void => {
     const delta = 3;
     for (let i: number = 0; i < delta; i++) {
       const color = neutralFillLayer(neutralPalette, neutralPalette.get(i), delta);
-      const resolved = neutralPalette.get(delta + i);
+      const resolved = neutralPalette.get(i - delta);
```

Comment:
GitHub can't possibly think that I'd want to comment on the line *4* above your change, but the logic looks correct and the text is now wrong. I'd say "should be _lighter or equal_ when the index of the backgroundColor is lower than the offset index"